### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by SaneMyburg**
Create desktop version to complete my project.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -369,3 +365,174 @@ h6 {
...
+}
+
+@media screen and (min-width: 768px) {
+  .app-container {
+    display: grid;
+    width: 100%;
+    /*background: #e5e5e5;
+    background-image: url(./images/Frame.svg);
+    background-repeat: no-repeat;*/
+  }
```
Comment (felixDev22) [inline style.css]: Please remove unused code. It will make your code look neat.

File: style.css [hunk 2]
```diff
@@ -369,3 +365,174 @@ h6 {
...
+    height: 100%;
+    border-top-right-radius: 60px;
+  }
+
+  .contact-me {
+    display: flex;
+    background-color: #fff;
+    justify-content: center;
+    /*max-width: 1440 px;*/
+    background-image: url(./images/contact-back.svg);
```
Comment (felixDev22) [inline style.css]: Please also see line 531. for better, cleaner, code - remove the unused code.

**Review by felixDev22 — CHANGES_REQUESTED** [review]
- [x] You are almost there...just small changes .
- [x] Overall you have done a good job.
- [x] Also, check for correct linters errors. Otherwise so far so good.

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @SaneMyburg,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:

## [Optional] suggestions:
 * Nothing to mention.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**